design/Commit/How a commit is done in FDB.md

```diff
@@ -0,0 +1,212 @@
+# How a commit is done in FDB
+
+## Overall description
```

Comment:
Please add a sentence:
This doc describes how commit is done in FDB 6.3+. 
The commit path in FDB 6.3 and before is documented in documentation/sphinx/source/read-write-path.rst (https://github.com/apple/foundationdb/pull/4099)